base-notebook/start.sh

```diff
@@ -168,12 +168,32 @@ if [ "$(id -u)" == 0 ] ; then
         PATH="${PATH}" \
         PYTHONPATH="${PYTHONPATH:-}" \
         "${cmd[@]}"
-        # Note on the purpose of "PATH=${PATH}":
-        #   In case "${cmd[@]}" is "bash", then PATH will be used by this bash shell.
-        #   However, PATH is irrelevant to how the above sudo command resolves the
-        #   path of "${cmd[@]}". Sudo's path resolution is done via the "secure_path"
-        #   variable set above in /etc/sudoers.d/path.
-
+        # Notes on how we ensure that the environment that this container is started
+        # with is preserved (except vars listen in JUPYTER_ENV_VARS_TO_UNSET) when
```

Comment:
```suggestion
        # with is preserved (except vars listed in JUPYTER_ENV_VARS_TO_UNSET) when
```

Reply:
Is `JUPYTER_ENV_VARS_TO_UNSET` a feature we have active?

Reply:
It is implemented, documented, and even tested.
I have no idea though if anyone uses it.

Reply:
It is implemented, documented, and even tested.
I have no idea though if anyone uses it